src/tests/run.cmd

```diff
@@ -176,7 +176,7 @@ if defined RunInUnloadableContext (
     set __RuntestPyArgs=%__RuntestPyArgs% --run_in_context
 )
 
-set NEXTCMD=python "%__RepoRootDir%\src\coreclr\tests\runtest.py" %__RuntestPyArgs%
+set NEXTCMD=python "%__RepoRootDir%\src\tests\run.py" %__RuntestPyArgs%
```

Comment:
fwiw, I wish this script was a much smaller wrapper around run.py. It looks like Jarret left a lot of stuff here because the Python version isn't used in the official build scenario. It would be nice to understand why that needs to be different, or if it was just left different because it was easier to leave it.

Reply:
I plan to look into both pairs of scripts (build.cmd / sh, run.cmd / sh) and start trimming them down with the ultimate goal to make most test-related logic directly runnable from the root build script per https://github.com/dotnet/runtime/issues/33183.

Reply:
FWIW I would prefer to remove dependencies on python if possible. It's just another build dependency that we don't depend on in the libraries and installer part of the repo. Just my 5c.

Reply:
I have already witnessed several lengthy and somewhat controversial discussions on the subject. I'm no big fan of Python myself but at the very least I want to separate that discussion from the mechanical task of removing the <code>src/coreclr/tests</code> folder. This PR should conclude the shuffle I started in the July infra rollout and my expectation is that, as the next step, I'll drill deeper into the details of the build and run scripts with the ultimate goal to make tests runnable from the root build script as I mentioned. I hope that, at least in this case, there'll be no Python involved and the root build logic will directly call into the MSBuild projects <code>src/tests/build.proj</code> and <code>src/tests/run.proj</code>.

Reply:
Yeah sorry for bringing this topic which is mostly unrelated, up here. I understand that higher up in the stack the python dependency is less an issue (like dotnet/performance) but in dotnet/runtime we might want to rethink if we want to depend on it. What you are saying is great, I'm a big fan of common paths with least amount of dependencies.